File: matiasmasso/Mat.NET/DTO/DTOGuidNode.cs
using System;
using System.Collections.Generic;

namespace DTO
{
    public class DTOGuidNode : DTOGuidNom
    {
        public List<DTOGuidNode> Items { get; set; }
        public DTOGuidNode(Guid oGuid, string sNom = "") : base(oGuid, sNom)
        {
            Items = new List<DTOGuidNode>();
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOGuidNode.vb
Public Class DTOGuidNode
    Inherits DTOGuidNom

    Property Items As List(Of DTOGuidNode)
    Public Sub New(oGuid As Guid, Optional sNom As String = "")
        MyBase.New(oGuid, sNom)
        _Items = New List(Of DTOGuidNode)
    End Sub
End Class
